v8pp/config.hpp.in

```diff
@@ -2,7 +2,11 @@
 #define V8PP_CONFIG_HPP_INCLUDED
 
 /// v8pp library version
-#define V8PP_VERSION "@PROJECT_VERSION@"
+#define V8PP_MAJOR @V8PP_MAJOR@
+#define V8PP_MINOR @V8PP_MINOR@
+#define V8PP_REVISION @V8PP_REVISION@
```

Comment:
Here should be used CMake component-wise version variables `PROJECT_VERSION_MAJOR`, `PROJECT_VERSION_MINOR`, `PROJECT_VERSION_PATCH`.

See details at https://cmake.org/cmake/help/v3.9/variable/PROJECT_VERSION.html#variable:PROJECT_VERSION